### DISCUSSION TITLE
Proposal submission

### DISCUSSION THREAD

**Discussion by kumariAstha** [github/india#692] [Students]
Where do I need to submit my proposal? I can't find any link or anything related to the same. Can someone guide me for the same?
(My profile is under review)

**Comment by rtpa25** [comment]
facing the same problem it seems till your profile is not verified you can't apply for proposals. And GitHub is taking way more time than required to review the profile

**Reply by kumariAstha** [reply]
Yeah , and today is the deadline to submit the proposal. So is there any other way to get any info related to the same?

**Reply by rtpa25** [reply]
No idea dude, even I am waiting for my profile to get verified in their reply mail yesterday they stated that it takes something like 48 hours for GitHub to review a profile
